### DISCUSSION TITLE
无法显示应用图标

### DISCUSSION THREAD

**Discussion by 30-Chen** [SPlayer-Dev/SPlayer#164] [Q&A]
系统版本：ubuntu24.04 LTS
软件版本：v2.0.7
问题描述：下载并安装`splayer_2.0.7_amd64.deb`后，**无法正常显示应用图标**，其他正常。

![截图 2024-05-05 15-24-08-01](https://github.com/imsyy/SPlayer/assets/167672604/47bf09e0-266e-47e7-bf3a-a560c6125c05)

**Comment by imsyy** [comment]
#95 已知问题
